test/parallel/test-child-process-fork-regr-gh-2847.js

```diff
@@ -1,3 +1,5 @@
+// Before https://github.com/nodejs/node/pull/2847 a child process trying
```

Comment:
Sorry, should be an issue, so:

`https://github.com/nodejs/node/pull/2847`->`https://github.com/nodejs/node/issues/2847`

Also can you order this as in [the guide](https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#test-structure)? Should look like:

```js
'use strict';
const common = require('../common');

// Before https://github.com/nodejs/node/issues/2847 a child process trying
// (asynchronously) to use the closed channel to it's creator caused a segfault.
 
const assert = require('assert');
```